scripts/install_python38.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) Granulate. All rights reserved.
+# Licensed under the AGPL3 License. See LICENSE.md in the project root for license information.
+#
+yum install -y openssl-devel bzip2-devel libffi-devel xz-devel wget
```

Comment:
As a first line in the script - add `set -euo pipefail` for a more resillient bash (e.g see `async_profiler_build_shared.sh`). This means you won't need things like `cd Python-3.8*/ || exit` because the script will fail if `cd` fails, so don't need the `exit`.